packages/phone-number-privacy/signer/test/integration/domain.test.ts

```diff
@@ -476,16 +486,21 @@ describe('domainService', () => {
           now: res1.body.status.now,
         },
       })
+      const eval1 = thresholdPoprfClient.unblindPartialResponse(
+        Buffer.from(res1.body.signature, 'base64')
+      )
+      expect(eval1.toString('base64')).toEqual(expectedEval)
 
       // submit identical request with nonce set to 1
-      const req2 = (await signatureRequest(undefined, 1))[0]
+      req1.options.nonce = defined(1)
```

Comment:
I'll change the name from `req1` -> `req`, though I intentionally want the request to be the exact same except for the nonce/signature. Otherwise, I think it creates room for the issue before which is that there's another difference in the requests (not just the nonce) that is sneakily causing success. Wdyt?